src/Converter/ConvertableEntities/Link.php

```diff
@@ -123,11 +125,18 @@ public function process( ?ConfluenceConverter $sender, DOMNode $match, DOMDocume
 			if ( $isMediaLink ) {
 				$replacement = $this->makeMediaLink( $linkParts );
 			} else {
-				$replacement = '[[' . implode( '|', $linkParts ) . ']]';
+				// Sometimes it could be that no label is set
+				if ( count( $linkParts ) > 1 ) {
+					$replacement = '[[' . implode( '|', $linkParts ) . ']]';
+				} else {
+					$labelParts = explode( ':', $linkParts[0] );
```

Comment:
Is is not using `basename()`, this in case `$linkParts[0] = "Some/sub/page"`, the label will be "Some/sub/page", opposed to "page" suggested in #50